apps/generic_terminal/unix_shell.talon

```diff
@@ -0,0 +1,26 @@
+tag: terminal
+tag: user.generic_unix_shell
+os: mac
+os: linux
+-
+# Generic Shell Commands. (General convension is to use the command name in "man" when possible)
+
+^change (directory|dir)$: "cd "
```

Comment:
We already have the `katie` command in `generic_terminal.talon`.

Reply:
see my comment above